production/loki-mixin/alerts.libsonnet

```diff
@@ -128,6 +128,21 @@
               |||,
             },
           },
+          {
+            alert: 'PromtailFileMissing',
+            expr: |||
+              sum (count by (path,instance,job) (promtail_file_bytes_total) unless count by (path,instance,job) (promtail_read_bytes_total)) > 0
```

Comment:
You could omit the `sum > 0` - the `count unless count` will return an empty set when all is well.  Alerts are the presence of timeseries matching the query.

Reply:
fixed!